mergify_engine/tests/unit/test_clients.py

```diff
@@ -68,6 +70,47 @@ async def test_client_installation_token(httpserver: httpserver.HTTPServer) -> N
     httpserver.check_assertions()
 
 
+@pytest.mark.asyncio
+async def test_client_installation_token_with_owner_id(
+    httpserver: httpserver.HTTPServer,
+) -> None:
+    with mock.patch(
+        "mergify_engine.config.GITHUB_API_URL",
+        httpserver.url_for("/")[:-1],
+    ):
+        httpserver.expect_request("/user/12345/installation").respond_with_json(
+            {
+                "id": 12345,
+                "target_type": "User",
+                "permissions": {
+                    "checks": "write",
+                    "contents": "write",
+                    "pull_requests": "write",
+                },
+                "account": {"login": "testing", "id": 12345},
+            }
+        )
+        httpserver.expect_request(
+            "/app/installations/12345/access_tokens"
+        ).respond_with_json(
+            {"token": "<app_token>", "expires_at": "2100-12-31T23:59:59Z"}
+        )
+
+        httpserver.expect_request(
+            "/", headers={"Authorization": "token <installation-token>"}
+        ).respond_with_json({"work": True}, status=200)
```

Comment:
It looks like this is really identical to the previous test. Could you refactor this so we follow [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)? 😄